### ISSUE TITLE
ModuleNotFoundError: No module named 'aeiou'

### ISSUE THREAD

**Issue by 1kari-s** [lks-ai/ComfyUI-StableAudioSampler#16]
In the console this error. ComfyUI cannot load StableAudioSampler nodes.
I tried installing through the manager and manually.
The stable_audio_tools folder is copied to python_embeded, it was scolded before.

**Comment by lks-ai** [comment]
That is super weird. Are you running ComfyUI Portable?

**Comment by 1kari-s** [comment]
Yes. I've already tried a complete reinstallation and it doesn't work.

**Comment by lks-ai** [comment]
Fix incoming ;)

**Comment by adydeejay** [comment]
Don't forget to update the huggingface links in the README.md file please. Thank you!

**Comment by GomuSkelly** [comment]
@1kari-s @adydeejay The information provided by @marius-jopen here: https://github.com/lks-ai/ComfyUI-StableAudioSampler/issues/30 might help you get this node running.

**Comment by ghost** [comment]
Fix doesn't work. 

1. `No module named 'aeiou'`
2. then `No module named 'librosa'`
3. then `No module named 'stable_audio_tools'`
4. then i got this 
```
      AttributeError: module 'pkgutil' has no attribute 'ImpImporter'. Did you mean: 'zipimporter'?
      [end of output]
```  

Why it's so hard for Python devs to test their code before releasing it for public? At least put exact package versions in requirements.txt to prevent conflicts - that is the basics. @lks-ai @newsbubbles @marius-jopen

**Comment by ghost** [comment]
Because modern python devs have learned on shitty web courses.
